Client/Frontend/Onboarding/OnboardingCardViewController.swift

```diff
@@ -288,6 +328,12 @@ class OnboardingCardViewController: UIViewController, Themeable {
         delegate?.showNextPage(viewModel.cardType)
     }
 
+    @objc
+    func linkButtonAction() {
+        viewModel.sendTelemetryButton(isPrimaryAction: false)
+        delegate?.showNextPage(viewModel.cardType)
```

Comment:
This line should be a `// TODO: https://mozilla-hub.atlassian.net/browse/FXIOS-5850` instead of an action we don't actually want.